compiler/src/dotty/tools/dotc/transform/ReifyQuotes.scala

```diff
@@ -144,6 +144,9 @@ class ReifyQuotes extends MacroTransformWithImplicits with InfoTransformer {
     /** We are in a `~(...)` context that is not shadowed by a nested `'(...)` */
     def inSplice = outer != null && !inQuote
 
+    /** We are not in a `~(...)` or a `'(...)` */
+    def isRoot = outer != null
```

Comment:
Hmm. I am confused. Isn't this root if the outer *is* `null` according to line 128 above? 

(also a typo in the comment of the description of the parameter `outer` )

Reply:
Actually it is a typo. It should be `isNotRoot`

Reply:
Or I will simplify it to an actual `isRoot` and add the not at call site.

Reply:
Good to go, then. Thanks!